### ISSUE TITLE
lstm_attention.py run error

### ISSUE THREAD

**Issue by sue2415535899** [cai-lw/image-captioning-chinese#3]
successfully run lstm.py
there is a error when run latm_attention.py in these codes:train_op = tf.train.AdamOptimizer().minimize(loss, var_list=train_vars).
error:Shapes (?, ?, 1073) and (?, ?, 512) are not compatible
my configuration:pythony3.5+tensorflow1.1+GPU
